### PR TITLE
[MIP] kimchi

### PR DESCRIPTION
**Pull request by mimoo**
The MIP introducing Kimchi

### CODE REVIEW

**Comment by yourbuddyconner** [comment]
This pull request has been mentioned on **MinaResearch**. There might be relevant details there:

https://forums.minaprotocol.com/t/kimchi-mip-a-proposed-upgrade-to-minas-proof-system/6004/1

File: mip-kimchi.md
Comment (imikushin) [inline mip-kimchi.md:1]: Should this be in the MIPS folder?

Comment (mimoo) [reply mip-kimchi.md:1]: thanks for catching this :)

**Comment by kantp** [comment]
Thanks @mimoo, this looks great!

One nitpick: I noticed this MIP does not have a `Test Cases` section, which the [README.md](https://github.com/MinaProtocol/MIPs/blob/999b5416c4987ee7254cc7872bbe03028af3fe6b/README.md) requires.

Can we add a section, or some comments here, on what kind of testing we're planning for including Kimchi in Mina?

**Pull request merged**